src/components/molecules/definitionitem/DefinitionItem.jsx

```diff
@@ -16,17 +16,11 @@ const DefinitionItem = ({ payload }) => {
 			</h1>
 
 			{drugInfoSummaryLink && (
-				<a className="definition-link" href={drugInfoSummaryLink.uri}>
-					<img
-						src="https://www.cancer.gov/images/btn-patient-info.gif"
-						alt="Patient Information"
-						title="Patient Information"
-						width="139"
-						height="20"
-						hspace="12"
-						border="0"
-						align="absmiddle"
-					/>
+				<a
+					className="paitent-information-button"
+					href={drugInfoSummaryLink.uri}
+					data-pos="top">
```

Comment:
What is the purpose of this custom attribute?

Reply:
removed.. oops type ahead..